### PR TITLE
(feat) Group Session Meta

### PR DESCRIPTION
**Pull request by ZacButko**
This adds concepts needed by the group session feature.

### CODE REVIEW

File: distro/configuration/concepts/group_session_meta.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:es,Short name:en,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1700
+e2559620-900b-4f66-ae41-0b9c4adfb654,,,Session Name,Nombre de sesi�n,Name,Nombre,Session Name,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline distro/configuration/concepts/group_session_meta.csv:2]: Change the date type to Text

File: distro/configuration/concepts/group_session_meta.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:es,Short name:en,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1700
+e2559620-900b-4f66-ae41-0b9c4adfb654,,,Session Name,Nombre de sesi�n,Name,Nombre,Session Name,Misc,N/A,,,,,,,,,,,
+ceaca505-6dff-4940-8a43-8c060a0924d7,,,Session Date,Fecha de la sesi�n,Date,Fecha,Session Date,Misc,Date,,,,,,,,,,,
+f1a2d58c-1a0e-4148-931a-aac224649fdc,,,Practitioner,Facultativo,Practitioner,Facultativo,Practitioner,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline distro/configuration/concepts/group_session_meta.csv:4]: Change the date type to Text

File: distro/configuration/concepts/group_session_meta.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:es,Short name:en,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1700
+e2559620-900b-4f66-ae41-0b9c4adfb654,,,Session Name,Nombre de sesi�n,Name,Nombre,Session Name,Misc,N/A,,,,,,,,,,,
+ceaca505-6dff-4940-8a43-8c060a0924d7,,,Session Date,Fecha de la sesi�n,Date,Fecha,Session Date,Misc,Date,,,,,,,,,,,
+f1a2d58c-1a0e-4148-931a-aac224649fdc,,,Practitioner,Facultativo,Practitioner,Facultativo,Practitioner,Misc,N/A,,,,,,,,,,,
+fa8fedc0-c066-4da3-8dc1-2ad8621fc480,,,Session Notes,Notas de la sesi�n,Notes,Notas,Session Notes,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline distro/configuration/concepts/group_session_meta.csv:5]: Change the date type to Text

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: distro/configuration/concepts/group_session_meta.csv (hunk 3 above)
Comment (ZacButko) [reply distro/configuration/concepts/group_session_meta.csv]: Ok changes pushed. Is this correct now?

**Comment by ZacButko** [comment]
@reagan-meant did I make the changes correctly?

**Review by reagan-meant — APPROVED** [review]

**Pull request merged**